include/netcdf.h

```diff
@@ -1966,6 +1966,19 @@ EXTERNL int nc_finalize();
 }
 #endif
 
+/* Define two hard-coded functionality-related (requested by
+   community developers) macros, but this is not going to be
+   standard practice.  Don't remove, they are depended upon
+   by downstream software. */
```

Comment:
If the downstream software that depends on this is netcdf-fortran, we can fix netcdf-fortran, right?